docs/content/_docs/config.md

```diff
@@ -624,9 +624,31 @@ settings: <map>
 
 The Elasticsearch client configuration to use.
 
+###### rest
+
+Connect using the High Level REST client. This is the preferred client for Elasticsearch and a replacement for the older transport client. This is compatabile with Elasticsearch node versions >= 7.5 and < 8.0.
```

Comment:
the high level rest client does not work with >= 8.0 cluster?

Reply:
This specific version is not guaranteed to be 100% compatible with an 8.x cluster. The REST client guarantees forward compatibility of all minor versions, but not majors.